.travis.yml

```diff
@@ -76,6 +78,8 @@ script:
 
     - etc/bin/test-packages
 
+    - gulp admin-css admin-js
```

Comment:
IMO we should modify Gulpfile.js and create new gulp task `gulp.task('travis', ['admin-js', 'admin-css', 'admin-img']);` then we can run gulp with travis parameter `gulp travis`.